### DISCUSSION TITLE
[Bug] Maccy fails to open when macOS Terminal is in focus

### DISCUSSION THREAD

**Discussion by balazs-ita-epam** [p0deje/Maccy#863] [Ideas]
Hi,

macOS Sonoma 14.8.1 MBPro, Maccy 1.0.0 (31)

When macOS Terminal app is the active window, activating Maccy with its shortcut fails: the dropdown list apperas for half a second then disappears. 

User is NOT able to select any Maccy entry for paste.

Please fix.

**Comment by p0deje** [comment]
This issue is now fixed in 2.0 beta, please upgrade to https://github.com/p0deje/Maccy/releases/tag/2.0.0.beta.13

**Reply by balazs-ita-epam** [reply]
Thanks, it fixed indeed!
